templates/infra/linodeCluster.yaml

```diff
@@ -11,3 +11,5 @@ spec:
     apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
     kind: LinodeVPC
     name: ${VPC_NAME:=${CLUSTER_NAME}}
+  network:
+    apiserverLoadBalancerPort: ${APISERVER_PORT:=6443}
```

Comment:
This is fine, not sure if we should set this by default in linodeCluster.yaml

Reply:
Yeah I think I'm going to remove it given the port doesn't seem to be (easily?) configurable for rke2 and k3s won't work without a hacky workaround to account for kustomize stripping the quotes from `KThreesServerConfig.spec.kthreesConfigSpec.serverConfig.httpsListenPort` when it shouldn't